c_src/crypto.c

```diff
@@ -154,6 +157,21 @@ static ERL_NIF_TERM evp_md_final(ErlNifEnv* env, erl_evp_md_ctx_t* erl_md_ctx)
     return ret;
 }
 
+static ERL_NIF_TERM rand_bytes(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[]) {
+    unsigned int bytes;
+    ERL_NIF_TERM ret;
+
+    if(!enif_get_uint(env, argv[0], &bytes)) {
```

Comment:
I don't know if we have a C-style guide, but I usually go with a space after `if` since it isn't a function call and below a space between `}` and `else`.